app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
```

Comment:
Fantastic job adding Device forms correctly 💯 

Please note that although styling is not required as this stage, you are still required to apply basic formatting styles to layout the elements according to the [sneak peak wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md).

### Log in

![image](https://github.com/Mike111222/BlogApp/assets/38088818/73d82419-d074-4c04-93d1-3274e81f567d)

Kindly take note that the login page has the following layout composition:

- The form is centered on the page and occupies about 80% of the view width
- The are no input labels but placeholders
- The login button is the same width as the input fields
- All other elements are removed, there's only email, password, and login button on the page

![image](https://github.com/Mike111222/BlogApp/assets/38088818/cf378e5f-11d6-4176-b0f2-c9ea396213fa)


### Sign up

Similarly, the same should be applied for the registration page.

![image](https://github.com/Mike111222/BlogApp/assets/38088818/adb1dea2-b71f-43fb-ba4b-165fa65d0dee)